cmd/skaffold/app/cmd/config/util.go

```diff
@@ -59,7 +60,10 @@ func resolveConfigFile() error {
 	return util.VerifyOrCreateFile(configFile)
 }
 
+// ReadConfigForFile reads the specified file and returns the contents
+// parsed into a Config struct.
 func ReadConfigForFile(filename string) (*Config, error) {
+	fmt.Printf("reading config for file %s\n", filename)
```

Comment:
ux-nit: this feels like a bit of clutter on the screen - can we make it available only for debug logging?